src/vs/workbench/contrib/scm/browser/dirtydiffDecorator.ts

```diff
@@ -590,7 +594,7 @@ KeybindingsRegistry.registerCommandAndKeybindingRule({
 	}
 });
 
-export class DirtyDiffController extends Disposable implements IEditorContribution {
+export class DirtyDiffController extends Disposable implements IEditorContribution, DirtyDiffContribution {
```

Comment:
Nit: Should this be just `DirtyDiffContribution` given that that extends `IEditorContribution`